tool/gravity/cli/system.go

```diff
@@ -47,6 +51,38 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func systemClusterInfo(env *localenv.LocalEnvironment) error {
```

Comment:
What's this used for?

Reply:
This wraps `kubectl cluster-info dump --output-directory=/path/to/dir` [here](https://github.com/gravitational/gravity/blob/dmitri/7.0.x/forward-port-6.3.x/lib/report/k8s.go#L41)